doc/slime.texi

```diff
@@ -3177,6 +3178,11 @@ Toggle exclusion of swank functions from the report.
 
 @end table
 
+@node SLIME Enhanced M-.
+@section SLIME Enhanced M-.
+@code{slime-mdot-fu} enables meta-point to jump to local variables bound with @code{let} and @code{let*},
```

Comment:
I think you should `M-x fill-paragraph` this. (And I think slime.texi assumes `fill-column` = 70.)